python/dask_cudf/dask_cudf/io/tests/test_json.py

```diff
@@ -82,6 +82,6 @@ def test_read_json_nested_experimental(tmp_path):
     kwargs = dict(orient="records", lines=True)
     with tmp_path / "data.json" as f:
         df.to_json(f, **kwargs)
-        actual = dask_cudf.read_json(f, engine="cudf_experimental", **kwargs)
+        actual = dask_cudf.read_json(f, engine="cudf", **kwargs)
```

Comment:
If `engine="cudf"` now works, then I'd also like to be testing the `actual = dask_cudf.read_json(f, **kwargs)` case here as well.

Reply:
Added this case too 👍